packages/blade/src/components/Tabs/__tests__/Tabs.native.test.tsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
```

Comment:
Add a test to make sure isLazy gets passed down properly?

Reply:
Unable to test that also, react-native-tabbar doesn't render it's own wrapper so their own props like `renderTabBar`, `navigationState`, `lazy` doesn't come up in snapshots.

Reply:
can't you do it like add isLazy and then do getbytext from a panel check if the text exist or not? if isLazy is passed you won't get any text rendered and vice-versa?

Reply:
Testability issues with the library: https://razorpay.slack.com/archives/G01B3LQ9H0W/p1698213664463119